applications/StructuralMechanicsApplication/custom_elements/truss_element_3D2N.hpp

```diff
@@ -158,6 +158,10 @@ namespace Kratos
             MatrixType& rMassMatrix,
             const ProcessInfo& rCurrentProcessInfo) override;
 
+        void CalculateConsistentMassMatrix(
+            MatrixType& rMassMatrix,
+            const ProcessInfo& rCurrentProcessInfo);

```

Comment:
```suggestion
            const ProcessInfo& rCurrentProcessInfo) const;

```

this must not modify members